Sources/SwiftTranslate/Models/AIModel.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Copyright © 2024 Hidden Spectrum, LLC.
+//
+
+import ArgumentParser
+import Foundation
+
+
+public enum AIModel: String, ExpressibleByArgument {
+    case gpt3_5Turbo = "gpt-3.5-turbo"
+    case gpt4o = "gpt-4o"
+    case gpt4o_mini = "gpt-4o-mini"
+    case gemini15 = "gemini-1.5-flash"
+    case gemini20 = "gemini-2.0-flash"
```

Comment:
If this is going to be unified, the command parse should guard against mismatched platform/models. Easiest thing would problem make `OpenAITranslator` / `GeminiTranslator` `init(...) throws`